docs/member/issuer-profile.md

```diff
@@ -0,0 +1,78 @@
+# Issuer Profile
+
+## 1. What is an Issuer?
+
+A legal entity that manages, registers, and sells securities to support its operations financially is an Issuer. There can be many types of Issuers - Corporations, Domestic or Foreign Governments and Investment trusts. Issuers develop securities such as Bonds, Debentures, G-Secs & NCDS. Issuers are legally responsible for reporting financial conditions, material developments, and other operational activities required by the jurisdiction's regulations.
+
+The AlgoBulls Issuer Profile gives you a detailed view of the issuer’s name, bonds available, transactions, average yield and more. Check the Issuer profile here.
+
+
+## 2. Issuer Profile Summary
+
+### i. Name
+
+This section displays the Issuer Name & Logo.
+
+### ii. Sector
+
+You can view the sector the Issuer belongs to here.
+
+### iii. Bio
+
+A detailed summary of the Issuer.
+
+[![Issuer Profile](imgs/issuer_profile_summary.png "Click to Enlarge or Ctrl+Click to open in a new Tab")](imgs/issuer_profile_summary.png)
+
+### iv. Number of Bonds
+
+View the number of Bonds available from the Issuer
```

Comment:
`View the number of Bonds available from the Issuer`

this line does not have a full stop (.) in the end. but other points above ad below have them

what is the consistency that we have followed for other pages? we keep the full stop or not? please change as per that

Reply:
Full stop added